installing/installing_gcp/installing-restricted-networks-gcp.adoc

```diff
@@ -0,0 +1,113 @@
+[id="installing-restricted-networks-gcp"]
+= Installing a cluster on GCP in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-networks-gcp
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Google Cloud Platform (GCP) by using infrastructure that you provide and an internal mirror of the installation release content.
+
+[IMPORTANT]
+====
+While you can install an {product-title} cluster by using mirrored installation
+release content, your cluster still requires internet access to use the GCP APIs.
+====
+
+The steps for performing a user-provided infrastructure install are outlined here. Several
+link:https://cloud.google.com/deployment-manager/docs[Deployment Manager] templates are provided to assist in
+completing these steps or to help model your own. You are also free to create
+the required resources through other methods; the templates are just an
+example.
+
+.Prerequisites
+
+* xref:../../installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a mirror registry on your bastion host]
+ and obtain the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the bastion host, use that computer
+to complete all installation steps.
+====
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall and plan to use telemetry, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+
+[id="installation-restricted-networks-gcp-user-infra-config-project"]
+== Configuring your GCP project
+
+Before you can install {product-title}, you must configure a Google Cloud
+Platform (GCP) project to host it.
+
+include::modules/installation-gcp-project.adoc[leveloffset=+2]
+include::modules/installation-gcp-enabling-api-services.adoc[leveloffset=+2]
+include::modules/installation-gcp-dns.adoc[leveloffset=+2]
+include::modules/installation-gcp-limits.adoc[leveloffset=+2]
+include::modules/installation-gcp-service-account.adoc[leveloffset=+2]
+include::modules/installation-gcp-permissions.adoc[leveloffset=+3]
+include::modules/installation-gcp-regions.adoc[leveloffset=+2]
+include::modules/installation-gcp-install-cli.adoc[leveloffset=+2]
+
+include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
+
+include::modules/installation-initializing.adoc[leveloffset=+2]
+
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
```

Comment:
Do we really need `proxy` added in this doc? Once proxy enabled, that means the whole cluster would have the access to Internet, then we don't need mirror registry any more. 
Unless customer have to use a mirror registry in their deployment and also need proxy to access some websites.

And for a cluster have no access to the Internet on GCP, I remember we need to ensure several addresses(to access some google services) must be accessible for the cluster when without proxy, I could sort them out later if we decide not to involve proxy.
@jianlinliu wdyt?